task.go

```diff
@@ -458,13 +458,14 @@ func (e *Executor) GetTaskList(filters ...FilterFunc) ([]*taskfile.Task, error)
 
 	// Compile the list of tasks
 	for i := range tasks {
-		task := tasks[i]
+		idx := i
+		task := tasks[idx]
 		g.Go(func() error {
 			compiledTask, err := e.FastCompiledTask(taskfile.Call{Task: task.Task})
```

Comment:
I also think that this `err` could be just `_` and remove the if that follows..

Reply:
Agreed, this is not needed.